### PR TITLE
583 fix sidebar scrollbar bugs

### LINKED ISSUE #583: Fix scrollbar on chrome / firefox
This happens if using windows / linux at route:

http://localhost:3000/routePath/0033,2013-06-17T00:00:00,1/?routes[0]=0033

### PR DESCRIPTION
**Pull request by Divinare**
Closes #583 

Every scrollbar should work now. Tested on linux chrome & firefox

### CODE REVIEW

File: src/components/sidebar/routePathView/routePathInfoTab/routePathInfoTab.scss [hunk 1]
```diff
@@ -2,33 +2,33 @@
 @import '../../../shared/styles/flexbox';
 @import '../../../shared/styles/form';
 
-.routePathTab {
+.routePathInfoTabView {
     display: flex;
     flex-direction: column;
+    overflow-y: auto;
```
Comment (kronken) [inline src/components/sidebar/routePathView/routePathInfoTab/routePathInfoTab.scss]: We early took a decision to have all Save buttons sticky at the bottom. This change breaks that, and requires the user scrolling.

File: src/components/sidebar/routesView/routesList.scss [hunk 2]
```diff
@@ -2,9 +2,9 @@
 .routesListView {
     display: flex;
     flex-direction: column;
+    overflow-y: auto;
```
Comment (kronken) [inline src/components/sidebar/routesView/routesList.scss]: This breaks sticky Save button

File: src/components/sidebar/routePathView/routePathInfoTab/routePathInfoTab.scss (hunk 1 above)
Comment (Divinare) [reply src/components/sidebar/routePathView/routePathInfoTab/routePathInfoTab.scss]: Fixed, save button moved to form's parent. TODO's added.

File: src/components/sidebar/routesView/routesList.scss (hunk 2 above)
Comment (Divinare) [reply src/components/sidebar/routesView/routesList.scss]: Fixed, save button moved to form's parent. TODO's added.

**Comment by Divinare** [comment]
Sticky save button should work. That button should be fixed at !591

**Pull request merged**